utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -15,4 +15,9 @@ annotation class SplashScreenWelcomeMsg
 
 val SPLASH_SCREEN_WELCOME_MSG = "splash_screen_welcome_msg"
 val SPLASH_SCREEN_WELCOME_MSG_DEFAULT_VALUE = false
-val SPLASH_SCREEN_WELCOME_MSG_VALUE = true
+
+@Qualifier
+annotation class SyncUpWorkerTimePeriod
+
+val SYNC_UP_WORKER_TIME_PERIOD = "sync_up_worker_time_period"
+val SYNC_UP_WORKER_TIME_PERIOD_DEFAULT_VALUE = 12
```

Comment:
Please indicate in the name the expected unit (e.g. seconds, minutes, etc.) since that's really important to ensure don't catastrophically reconfigure this in the future.

Reply:
Done